internal/util/projutil/project_util.go

```diff
@@ -315,7 +315,7 @@ func CheckGoModules() error {
 	}
 	if !goModOn {
 		return fmt.Errorf(`using go modules requires GO111MODULE="on", "auto", or unset.` +
-			` More info: https://sdk.operatorframework.io/docs/golang/quickstart/#a-note-on-dependency-management`)
+			` More info: https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/`)
```

Comment:
```suggestion
			` More info: https://v0-19-x.sdk.operatorframework.io/docs/golang/quickstart/#a-note-on-dependency-management`)
```

Reply:
we need to point out for the doc of this branch 0.19. Also, this is the places where has the info ^